src/Tribe/Widget/Widget_Interface.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace Tribe\Widget;
+
+use Tribe\Events\Views\V2\View_Interface;/**
```

Comment:
Not in Tribe Common (currently)

Reply:
@bordoni do you have some insight how we might get around this?

Reply:
Views needs to come, bring it home @bordoni

Reply:
Moved all the views related coding to an abstract class in TEC.